aws-observability-terraform/versions.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+    }
+    null = {
+      source = "hashicorp/null"
+    }
+    sumologic = {
+      source = "terraform-providers/sumologic"
```

Comment:
Better to lock the lowest version for all providers i.e. sumo, aws and null

Reply:
Done.